### PR TITLE
update links; tweak homepage hero and call to action

### PR DESCRIPTION
**Pull request by bnewbold**
- add links to github discussions (original motivation)
- promote sandbox instead of waitlist
- deep link in to overview and specs
- slightly less Bluesky PBC oriented (though still prominently mentioned/linked)

### CODE REVIEW

**Review by pfrazee — APPROVED** [review]
sgt,

File: content/community/quick-start.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Quick Start Guide
+title: Typescript Quick Start
```
Comment (emilyliu7321) [inline content/community/quick-start.md:2]:
```suggestion
title: TypeScript Quick Start Guide
```

File: content/docs.yml [hunk 2]
```diff
@@ -42,11 +42,11 @@ lexicons:
   - name: com.atproto
     href: /lexicons/com-atproto
 community:
-  - name: Quick Start Guide
+  - name: Typescript Quick Start
```
Comment (emilyliu7321) [inline content/docs.yml:45]:
```suggestion
  - name: Typescript Quick Start Guide
```

File: components/cta.js [hunk 3]
```diff
@@ -5,15 +5,15 @@ export default function CTA() {
     <div className="bg-blue-700">
       <div className="mx-auto max-w-2xl py-16 px-4 text-center sm:py-20 sm:px-6 lg:px-8">
         <h2 className="text-3xl font-bold tracking-tight text-white sm:text-4xl">
-          <span className="block">Join the Bluesky private beta.</span>
+          <span className="block">Join the Federated Sandbox</span>
         </h2>
         <p className="mt-4 text-lg leading-6 text-blue-200">
-          The AT Protocol will launch soon.
+          Self-hosted, no waitlist or invite code required. The production network will open to federation soon.
```
Comment (emilyliu7321) [inline components/cta.js:11]:
```suggestion
          No waitlist or invite code required to join this self-hosted sandbox environment. The production network will open to federation soon.
```

**Review by emilyliu7321 — APPROVED** [review]
few tweaks but otherwise lgtm 👍

**Pull request merged**